source/declarations.tex

```diff
@@ -5367,12 +5367,11 @@
 UTF-32 string literal, or
 wide string literal,
 respectively, or by an appropriately-typed \grammarterm{string-literal} enclosed in
-braces\iref{lex.string}.
+braces.
 \indextext{initialization!character array}%
-Successive
-characters of the
-value of the \grammarterm{string-literal}
-initialize the elements of the array.
+The elements of the array are initialized
```

Comment:
@jensmaurer  Please look at this [comment](https://github.com/cplusplus/draft/issues/5103#issuecomment-974738921). `values of the sequence of code unit values` sounds a bit vague; what value is that?